src/smart-components/Message/components/TextFragment/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { UserMessage } from '@sendbird/chat/message';
+
+import { TOKEN_TYPES, Token } from '../../utils/tokens/types'
+import { useMessageContext } from '../../context/MessageProvider';
+import { keyGenerator } from '../../utils/tokens/keyGenerator';
+import MentionLabel from '../../../../ui/MentionLabel';
+import { USER_MENTION_PREFIX } from '../../consts';
+import LinkLabel from '../../../../ui/LinkLabel';
+import { LabelTypography } from '../../../../ui//Label';
+
+export type TextFragmentProps = {
+  tokens: Token[];
+}
+
+export default function TextFragment({
+  tokens,
```

Comment:
It seems good to have one of both way (1) set default option or (2) use optional chaining

tokens?.map

Reply:
will do ~